src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -47,6 +47,8 @@ def get(self, request, project):
 
         for config in configs:
             if not filepath.startswith(config.stack_root):
+                result["config"] = serialize(config, request.user)
```

Comment:
@MeredithAnya we have this line twice, we can just move this to the top of the for loop.